js/mmtf.js

```diff
@@ -1,6 +1,8 @@
 (function (global, factory) {
   (factory((global['MMTF'] = global.MMTF || {})));
-}(this, function (exports) { 'use strict';
+
+}(window, function (exports) { 'use strict';
```

Comment:
I don't think we should hardcode global into being window no matter what.  Instead it might better to set
`global = global || window;`
although what really needs to happen is properly modularize everything.